third_party/2/google/protobuf/any_pb2.pyi

```diff
@@ -0,0 +1,25 @@
+# @generated by generate_proto_mypy_stubs.py.  Do not edit!
```

Comment:
should we snip these lines? they refer to something not in typeshed

Reply:
Happy to snip them out.